source/server/http/admin.h

```diff
@@ -220,6 +220,11 @@ class AdminImpl : public Admin,
                                  bool used_only,
                                  const absl::optional<std::regex> regex = absl::nullopt,
                                  bool pretty_print = false);
+  static absl::optional<std::regex> filterParam(Http::Utility::QueryParams params) {
```

Comment:
Minor thing but I'd probably put this in the anonymous namespace in the .cc file, no reason to put this on the class

Reply:
updated